src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,19 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { Todo } from '../../types';
+
+interface TodoInfoProps {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      <UserInfo user={todo.user} />
```

Comment:
Ensure that the `todo.user` object passed to `UserInfo` is always valid. If there's a chance that `todo.user` could be undefined or null, consider adding a check or default value to prevent runtime errors.